phpunit/class-block-templates-test.php

```diff
@@ -114,7 +114,7 @@ function test_gutenberg_build_template_result_from_file() {
 		$this->assertEquals( 'header', $template_part->title );
 		$this->assertEquals( '', $template_part->description );
 		$this->assertEquals( 'wp_template_part', $template_part->type );
-		$this->assertEquals( WP_TEMPLATE_PART_AREA_HEADER, $template_part->area );
+		$this->assertEquals( 'header', $template_part->area );
```

Comment:
Can you explain the change here?

Reply:
Oh thanks for catching that, that wasn't supposed to happen!  (updated)

I was testing out something unrelated to this PR and it must have snuck its way in to a commit - someone had suggested that perhaps both the defined constant and template->area were coming in as `undefined` in the php unit tests. 🤣